samples/Notifications.sarif

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample illustrates tool configuration and execution notifications."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples",
+          "rules": [
+            {
+              "id": "TUT1201",
+              "defaultConfiguration": {
+                "level": "error"
+              },
+              "messageStrings": {
+                "default": {
+                  "text": "The statement does not end with a semicolon."
+                }
+              }
+            }
+          ],
+          "notifications": [
+            {
+              "id": "TUTN9001",
+              "name": "unknown-rule",
+              "defaultConfiguration": {
+                "level": "warning"
+              },
+              "shortDescription": {
+                "text": "This notification is triggered when the user supplies a command line argument to enable or disable a rule that does not exist."
+              },
+              "messageStrings": {
+                "disabled": {
+                  "text": "The rule '{0}' cannot be disabled because it does not exist."
```

Comment:
>The rule '{0}' cannot be disabled because it does not exist." [](start = 27, length = 61)

Suggest removing 'The rule' and referencing rule later. IO, you would prefer to see: Same pattern for enabled.

'{0}'' cannot be disabled because this rule does not exist.